storage-proofs/src/circuit/drgporep.rs

```diff
@@ -545,7 +545,7 @@ mod tests {
         let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
 
         let nodes = 12;
-        let degree = BASE_DEGREE;
+        let m = BASE_DEGREE;
```

Comment:
it should hold that `graph.degree() == BASE_DEGREE`, so the constants should be adjusted